x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -562,6 +564,47 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     return await this.baseClient.removeReferencesTo(type, id, options);
   }
 
+  public async openPointInTimeForType(
+    type: string | string[],
+    options: SavedObjectsOpenPointInTimeOptions
+  ) {
+    try {
+      const args = { type, options };
+      await this.ensureAuthorized(type, 'open_point_in_time', options?.namespace, {
+        args,
+        requireFullAuthorization: false,
+      });
+    } catch (error) {
+      this.auditLogger.log(
+        savedObjectEvent({
+          action: SavedObjectAction.OPEN_POINT_IN_TIME,
+          error,
+        })
+      );
+      throw error;
+    }
+
+    this.auditLogger.log(
+      savedObjectEvent({
+        action: SavedObjectAction.OPEN_POINT_IN_TIME,
```

Comment:
Sorry, should have mentioned in the first review: we will need [doc updates](https://github.com/elastic/kibana/blob/25f16db4d9791a8943b386cccb8680d70bc6bf54/docs/user/security/audit-logging.asciidoc#L39) to capture these new audit actions as well